docs/bigquery-usage.rst

```diff
@@ -429,3 +443,112 @@ Poll until the job is complete:
    'done'
    >>> job.ended
    datetime.datetime(2015, 7, 23, 9, 30, 21, 334792, tzinfo=<UTC>)
+
+Exporting data (async)
+~~~~~~~~~~~~~~~~~~~~~~
+
+Start a job exporting a table's data asynchronously to a set of CSV files,
+located on GCloud Storage.  First, create the job locally:
```

Comment:
We never resolved the `GCloud` vs. `gcloud` vs. `Gcloud` discussion before. Remind me not to LGTM until we at least discuss.